### PR TITLE
Updating lost gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
898_last_comms
525_last_comms

**Comment by sauuyer** [comment]
@cdobs What should be referenced to verify these annotations? I haven't reviewed glider statuses before.

**Comment by cdobs** [comment]
> @cdobs What should be referenced to verify these annotations? I haven't reviewed glider statuses before.

Hi @sauuyer sorry my message in slack was a little misleading. For this PR I am just updating the deploy csvs for 2 gliders that were lost. In cases where they are lost we enter UNRECOVERED for the recover CUID and enter the last known communication time for the recovery time. To confirm the last known communication time I posted some screenshots from our piloting notes in the PR comments

File: deployment/GI05MOAS-G0898_Deploy.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
-AR76-03,,,,GI05MOAS-G0898-02-DOSTAM000,1,1,2023-08-30T15:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-DOSTAM-00685,59.8827,-39.0366,,1000,1000,top mount,,
-AR76-03,,,,GI05MOAS-G0898-04-CTDGVM000,1,1,2023-08-30T15:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-CTDGVM-09678,59.8827,-39.0366,,1000,1000,,,
\ No newline at end of file
+AR76-03,,UNRECOVERED,,GI05MOAS-G0898-00-ENG000000,1,1,2023-08-30T15:00:00,2024-02-01T21:04:44,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGCON-GLDROO-00898,59.8827,-39.0366,,1000,1000,,,
+AR76-03,,UNRECOVERED,,GI05MOAS-G0898-01-FLORDM000,1,1,2023-08-30T15:00:00,2024-02-01T21:04:44,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-FLORDM-06989,59.8827,-39.0366,,1000,1000,,,
+AR76-03,,UNRECOVERED,,GI05MOAS-G0898-02-DOSTAM000,1,1,2023-08-30T15:00:00,2024-02-01T21:04:44,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-DOSTAM-00685,59.8827,-39.0366,,1000,1000,top mount,,
+AR76-03,,UNRECOVERED,,GI05MOAS-G0898-04-CTDGVM000,1,1,2023-08-30T15:00:00,2024-02-01T21:04:44,CGVGI-05MOAS-00000,CGVEH-GLDROO-00898,CGINS-CTDGVM-09678,59.8827,-39.0366,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-G0898_Deploy.csv:5]: looks good.  RIP glider 898

File: deployment/GI05MOAS-GL525_Deploy.csv [hunk 2]
```diff
@@ -3,7 +3,7 @@ AR35-05,,AJ20200506,,GI05MOAS-GL525-00-ENG000000,2,1,2019-08-06T17:23:00,2020-05
...
-AR69-01,,,,GI05MOAS-GL525-02-DOSTAM000,3,1,2022-06-28T16:58:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-DOSTAM-00569,59.81738,-39.11595,,1000,1000,,,
-AR69-01,,,,GI05MOAS-GL525-04-CTDGVM000,3,1,2022-06-28T16:58:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-CTDGVM-09232,59.81738,-39.11595,,1000,1000,,,
\ No newline at end of file
+AR69-01,,UNRECOVERED,,GI05MOAS-GL525-00-ENG000000,3,1,2022-06-28T16:58:00,2023-04-16T04:43:03,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGCON-GLDROO-00525,59.81738,-39.11595,,1000,1000,,,
+AR69-01,,UNRECOVERED,,GI05MOAS-GL525-01-FLORDM000,3,1,2022-06-28T16:58:00,2023-04-16T04:43:03,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-FLORDM-03579,59.81738,-39.11595,,1000,1000,,,
+AR69-01,,UNRECOVERED,,GI05MOAS-GL525-02-DOSTAM000,3,1,2022-06-28T16:58:00,2023-04-16T04:43:03,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-DOSTAM-00569,59.81738,-39.11595,,1000,1000,,,
+AR69-01,,UNRECOVERED,,GI05MOAS-GL525-04-CTDGVM000,3,1,2022-06-28T16:58:00,2023-04-16T04:43:03,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-CTDGVM-09232,59.81738,-39.11595,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL525_Deploy.csv:9]: looks good. RIP glider 525

**Review by rtravis-whoi — APPROVED** [review]

**Review by sauuyer — APPROVED** [review]
Looks good.

**Pull request merged**